openlane/steps/step.py

```diff
@@ -830,7 +836,8 @@ def start(
 
         state_in_result = self.state_in.result()
 
-        rule(f"{self.long_name}")
+        if not _no_rule:
+            rule(f"{self.long_name}")
```

Comment:
else: info() ? 👼🏿

Reply:
shit i forgot about htis one

Reply:
lets defer it for the logging overhaul ok